### ISSUE TITLE
[Question] see the zone that is affected

### ISSUE THREAD

**Issue by Goten87** [Xalcon/TorchMaster#206]
is there a way to see what range the mega tourch has on in the game (im now always counting 64 blocks from the tourch to see where i leads) but is there a way to to toggle a way to see it with a keybind or command like for the light levels F7

**Comment by Xalcon** [comment]
No, its not possible at the moment. It's on my todo list, but its not trivial to implement due to the high range of the torch (especially on low render distance, when chunks wont get rendered if you are too far away)

**Comment by Goten87** [comment]
maby include something with journymap (if people have it where you can show the area) sins it has intergration for mods

**Comment by Xalcon** [comment]
It's been a while, but I've started implementing a simple renderer that shows the volume of any Mega Torch or Dread Lamp after interacting with it in the world (Either Empty Hand or Vanilla Dye). Shift+Rightlick with an Empty Hand to disable the renderer for a given torch again.

This is currently only implemented for 1.20.1 Forge/Neoforge (Torchmaster 20.1.7). I'll test it for a few days before porting it to 1.21

**Comment by Seblor** [comment]
Any update on this feature ?

**Comment by AceDetectivePuff** [comment]
This would be really handy if you decide to port it over, I was just over here to see if a feature like this exists. I've been using these torches with my Minecolony, and I was hoping to figure out the best placement for coverage

**Comment by Xalcon** [comment]
Implemented in 1.21.1, will be released hopefully soon. I want to add a few more features first.
